src/ui/layout/row-layout.svelte

```diff
@@ -31,7 +32,7 @@
   }
 
   .rcb-column.gap-medium {
-    gap: var(--rc-spacing-gapMedium-mobile);
+    gap: var(--rc-spacing-gapXLarge-mobile);
```

Comment:
Closest to design spec agreed with Iga, `.rcb-column.gap-medium` isn't used currently so this doesn't affect any existing styles. Alternatively could rename as gap-xlarge and gap-large as gap-xxlarge (plus just remove gap-small)?